forum/notifications/models.py

```diff
@@ -1,18 +1,26 @@
 from django.db import models
-from django.contrib.auth.models import User
+from users.models import CustomUser
 
 
 class NotificationType(models.Model):
     name = models.CharField(max_length=100, unique=True)
     description = models.TextField(blank=True)
 
     def __str__(self):
-        return f"Preferences for {self.user.username if self.user else 'Unknown User'}"
+        return f"Notification type(name={self.name}, description={self.description}"
 
 
+class NotificationCategory(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return f"Notification category(name={self.name}, description={self.description}"
```

Comment:
Looks like the string is missing a closing parenthesis. Ensure the string representation is correctly formatted.

Reply:
Done ✅